mangaki/mangaki/settings.py

```diff
@@ -109,10 +109,10 @@
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'NAME': config.get('pgsql', 'DB_NAME', fallback='mangaki'),
-        'USER': config.get('pgsql', 'DB_USER', fallback='django'),
-        'PASSWORD': config.get('secrets', 'DB_PASSWORD'),
-        'HOST': config.get('pgsql', 'DB_HOST', fallback='127.0.0.1'),
-        'PORT': '5432',
+        'USER': config.get('pgsql', 'DB_USER', fallback=''),
```

Comment:
Is there any rationale behind this? `django` looks like a sane default to me

Reply:
Yes, allowing people to use PostgreSQL local auth, as specified in the commit message. USER, HOST and PASSWORD have to be empty for it to work.